.github/workflows/gh-pages.yml

```diff
@@ -19,6 +19,12 @@ on:
         required: false
         default: true
 
+      update_search_index:
+        description: Update the search index by triggering the Algolia crawler
+        type: boolean
+        required: false
+        default: false
```

Comment:
I would add a comment to tell that:

- The default must be false (because of the open-source plan limitations)
- This action is triggered from https://github.com/thin-edge/tedge-docs (this is not obvious)
  and enable as true from there.

Reply:
Resolved by d390ed00b2dcd5c6eaf397866b187f1eb22f08e5